.circleci/config.yml

```diff
@@ -24,3 +24,34 @@ jobs:
       - run:
           name: Test images
           command: tests/run.sh
+
+  trigger_dockerhub:
+    docker:
+      - image: integratedexperts/ci-builder
```

Comment:
Can we change this to `singledigital/bay-ci-builder:edge` ?

Reply:
Yeah - I was actually thinking about doing that too after I noticed that we had one set up - will fix it up.